core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,11 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  .popover {
```

Comment:
@seanlip  I was wondering about the merging too. If this PR goes first @shivanXI has to pull the changes before his merge occurs. If his goes first I should pull the changes. Either of one PR must re-write the whole code after merge, Is it right?